File: test-rest-api/src/com/inform8/services/auth/LoginService.java
package com.inform8.services.auth;

import com.google.gson.Gson;
import com.google.gson.JsonSyntaxException;
import com.inform8.entities.LoginResponse;
import com.inform8.entities.Response;
import com.inform8.services.Service;
import com.inform8.services.StaticConfig;

public class LoginService extends Service {
	public Response login(String username, String password){
		Response data = new Response();
		this.postData("Username", username);
		this.postData("Password", password);
		String output = "";
		try{
			output = this.post(StaticConfig.host+"/authenticate.php?call=login");
			Gson gsonData = new Gson();
			System.out.println(output);
			data = gsonData.fromJson(output, LoginResponse.class);
		}catch(JsonSyntaxException je){
			System.out.println(output);	
		}
		return data;
	}
}

File: retail-sas-client/app/transact/auth/LoginService.js
Ext.define("RS.transact.auth.LoginService", {
	extend:"RS.transact.Service",
	contructor:function(){
		this.url = baseUrl+"/authenticate.php?call=login"
	},
	run:function(username, password){
		Ext.Ajax.request({
			url:this.url,
			success:this.callback,
			params:{
				username:username,
				password:password
			}
		});
	}
});